### ISSUE TITLE
Lost  "

### ISSUE THREAD

**Issue by h2ws** [curlconverter/curlconverter#227]
curl 'www.google.com' \

  -H 'Cookie: SIG="K6A6AOTsRrhfdcCp4kzN7DUQG45joGLhZNLWdhd6+gj2ziy52fNueIROI8qz1IDIipVVdftdjK06QngVbkS+oA=="' 

coverted(python) ==>

import requests

cookies = {
    'SIG': 'K6A6AOTsRrhfdcCp4kzN7DUQG45joGLhZNLWdhd6+gj2ziy52fNueIROI8qz1IDIipVVdftdjK06QngVbkS+oA==',
}

headers = {
}

response = requests.get('http://www.google.com', headers=headers, cookies=cookies)

**Comment by verhovsky** [comment]
This issue makes sense to me. The converted output should have `'SIG': '"K6...oA=="',` but it removes the extra `"` around the value `'SIG': 'K6...oA==',` leading to a different result being sent:

```sh
curl -H 'Cookie: SIG="K6A6AOTsRrhfdcCp4kzN7DUQG45joGLhZNLWdhd6+gj2ziy52fNueIROI8qz1IDIipVVdftdjK06QngVbkS+oA=="' localhost:8888
```

```sh
$ nc -l localhost 8888
GET / HTTP/1.1
Host: localhost:8888
User-Agent: curl/7.77.0
Accept: */*
Cookie: SIG="K6A6AOTsRrhfdcCp4kzN7DUQG45joGLhZNLWdhd6+gj2ziy52fNueIROI8qz1IDIipVVdftdjK06QngVbkS+oA=="
```

vs. the converted result

```py
>>> import requests
>>> 
>>> cookies = {
... 'SIG': 'K6A6AOTsRrhfdcCp4kzN7DUQG45joGLhZNLWdhd6+gj2ziy52fNueIROI8qz1IDIipVVdftdjK06QngVbkS+oA==',
... }
>>> 
>>> headers = {
... }
>>> 
>>> response = requests.get('http://localhost:8888', headers=headers, cookies=cookies)
```

```sh
$ nc -l localhost 8888
GET / HTTP/1.1
Host: localhost:8888
User-Agent: python-requests/2.25.1
Accept-Encoding: gzip, deflate
Accept: */*
Connection: keep-alive
Cookie: SIG=K6A6AOTsRrhfdcCp4kzN7DUQG45joGLhZNLWdhd6+gj2ziy52fNueIROI8qz1IDIipVVdftdjK06QngVbkS+oA==
```

**Comment by verhovsky** [comment]
Fixed in #347
